### PR TITLE
Update AccountAutomations.cls

### PR DESCRIPTION
**Pull request by blessumkevin**
Please check this out. Made the Boston change you wanted.

### CODE REVIEW

**Review by dustinlevi — CHANGES_REQUESTED** [review]
Please do the same to Line 11 & re-submit.

**Comment by blessumkevin** [comment]
> Please do the same to Line 11 & re-submit.

Finished. Please double check again. Thanks.

File: classes/AccountAutomations.cls [hunk 1]
```diff
@@ -8,13 +8,13 @@ public class AccountAutomations {
     }
     public static Account setDefaultBillingAddress(Account a){
         a.billingstreet = 'Basauri, 6'; 
-        a.billingstate = 'Madrid';
```
Comment (dustinlevi) [inline classes/AccountAutomations.cls:11]: Thanks for fixing this!

**Review by dustinlevi — APPROVED** [review]
Looks good!

**Pull request merged**